daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -374,7 +375,7 @@ func checkLogicSigPass(t *testing.T, response *generated.DryrunResponse) {
 	}
 }
 
-func checkAppCallPass(t *testing.T, response *generated.DryrunResponse) {
+func checkAppCallResponse(t *testing.T, response *generated.DryrunResponse, responseString string) {
```

Comment:
Personal opinion: in a nice strongly typed language like Go, you can just use `response` here.

Reply:
Could you elaborate what you mean here?

Reply:
just to name the argument "response" rather than "responseString". I hadn't notice there was already an argument with that name.  I guess I might call it "msg", "lastMsg", "passFail"... or maybe make it a boolean and internally use "PASS" or "REJECT".  But it's no big deal if none of those options seem appealing.

Reply:
Gotcha makes sense, not opposed to `msg` so I renamed it 😄